main/src/cgeo/geocaching/maps/PositionHistory.java

```diff
@@ -41,6 +66,7 @@ public void rememberTrailPosition(final Location coordinates) {
             return;
         }
 
+        saveToStorage(coordinates);
```

Comment:
Essentially I would prefer to save the history only when the map activity is stopped, but when I looked through the implementation, it seemed not too easy to get that triggered in a safe fashion, so we might leave it like it is implemented here for the moment.